packages/HilltopCrawler/src/main/resources/db/migration/V0001__Hilltop_Indexing.sql

```diff
@@ -1,26 +1,26 @@
 CREATE TABLE hilltop_sources
 (
-    id         SERIAL      NOT NULL,
-    council_id INT         NOT NULL,
-    hts_url    VARCHAR     NOT NULL,
-    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
-    updated_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    id            SERIAL      NOT NULL,
+    council_id    INT         NOT NULL,
+    hts_url       VARCHAR     NOT NULL,
+    configuration JSONB       NOT NULL,
+    created_at    TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    updated_at    TIMESTAMPTZ NOT NULL DEFAULT NOW(),
     PRIMARY KEY (id),
     UNIQUE (council_id, hts_url)
 );
 
 CREATE TABLE hilltop_fetch_tasks
 (
     id                 SERIAL      NOT NULL,
-    council_id         INT         NOT NULL,
+    source_id          INT         NOT NULL,
     request_type       VARCHAR     NOT NULL,
-    base_url           VARCHAR     NOT NULL,
-    query_params       VARCHAR     NOT NULL,
-    state              VARCHAR     NOT NULL,
+    fetch_url          VARCHAR     NOT NULL,
     next_fetch_at      TIMESTAMPTZ NOT NULL DEFAULT NOW(),
```

Comment:
Should `next_fetch_at`be indexed since it's used to order when fetching the next task?

Reply:
Whoops, yes it should be!